src/relay/src/createRelayStore.js

```diff
@@ -2,15 +2,17 @@
 
 import { RecordSource, Store } from 'relay-runtime';
 
-import type { RecordMap } from './runtimeTypes.flow';
+import type { RecordMap, Scheduler, OperationLoader, GetDataID } from './runtimeTypes.flow';
 
 type Options = {|
-  +gcReleaseBufferSize?: ?number,
+  gcScheduler?: ?Scheduler,
+  operationLoader?: ?OperationLoader,
+  UNSTABLE_DO_NOT_USE_getDataID?: ?GetDataID,
```

Comment:
Can we hide this property or it would cause some Flow errors somewhere else?

Reply:
It would cause flow-error yes.